File: usercode.py
import jwt
def handle(data):
   

    payload_data = {
      "iat": data["now"],
      "exp": data["future"],
      "jti": data["jti"],
      "application_id": data["application_id"]
	 }


    key = data["key"]
    token = jwt.encode(
      payload=payload_data,
      key=key,
      algorithm='HS256'
    )
   
    data["hello"] = "Hello world!"
    data["token"] = token
    return data

File: usercode.js
const jwt = require('jsonwebtoken');


module.exports = (data) => {

	var privateKey = fs.readFileSync('private.key');
	var token = jwt.sign({ foo: 'bar' }, privateKey, { algorithm: 'RS256'});
    
    data.token = token;
    return data;
};
